AndrewTehan/hometask-2/get_middle_charachter.rb

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env ruby
+# frozen_string_literal: true
+
+ENTER_STRING = 'Enter string: '
+ERROR_MESSAGE = "String can't be blank"
+EMPTY_STRING = ''
+EXIT = 'exit!'
+
+def run_cli
+  print ENTER_STRING
+  string = gets.chomp
+
+  case string
+  when EXIT then exit
+  when EMPTY_STRING then puts ERROR_MESSAGE
+  else puts get_middle(string)
+  end
+
+  run_cli
+end
+
+def return_by_index(string, index)
```

Comment:
`return_` in methods' names seems weird a bit. One of the main purposes of methods is to do and **return** smth. So I'd say that it's not necessary to put such meanless words in naming from this point of view.